webapps/api/oauth2/tokenview.py

```diff
@@ -47,7 +49,8 @@ def post(self, request, *args, **kwargs):
         """
         Handles token post
         """
-        logger = Logger('api')
+        expiration_user = Configuration.get('ovs/framework/api/oauth|expiration_user', default=OAuth2Toolbox.EXPIRATION_USER)
+        expiration_client = Configuration.get('ovs/framework/api/oauth|expiration_user', default=OAuth2Toolbox.EXPIRATION_CLIENT)
```

Comment:
incorrect key should be `expiration_client`

Reply:
I will change the time out in the unit tests as this should have been picked up by them :p